disk/service_checks.json

```diff
@@ -0,0 +1,11 @@
+[
+    {
+        "agent_version": "5.22.3",
```

Comment:
This should be `6.5.0` since it will be released in the next version of the agent.

Reply:
sorry didn't know what that version was for